core/model-api/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.rdf4j</groupId>
+		<artifactId>rdf4j-core</artifactId>
+		<version>3.5.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>rdf4j-model-api</artifactId>
+	<name>RDF4J: Model API</name>
+	<description>RDF model interfaces.</description>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-jar-plugin</artifactId>
+				<executions>
+					<execution>
+						<goals>
+							<goal>test-jar</goal>
```

Comment:
I've added this to make sure the test-jar gets produced. I originally wanted to move your abstracts tests to the rdf4j-model-testsuite, but that then results in a circular dependency between rdf4j-model-api and rdf4j-model-testsuite. So let's go with the test-jar approach instead.